source/_posts/2015-03-23-artsy-technology-stack-2015.markdown

```diff
@@ -0,0 +1,56 @@
+
+<center><img src='/images/2015-03-23-artsy-technology-stack-2015/developers.png'></center>
+
+Our partners upload artworks, artists and other metadata via a home-grown content-management system built entirely from scratch in 2014 on Ruby-on-Rails. This was a deliberate "boring" choice that continues to serve us very well. We have adopted a common model for admin-type apps with a shared component library and a reusable UI, all implemented as Rails engines. Using these components we are able to quickly and easily compose beautiful and useful applications - we have built dedicated systems to manage fairs and auctions. We standardized on related services as well - for example, our customers interact with us via [Intercom](https://www.intercom.io).
+
+<center><img src='/images/2015-03-23-artsy-technology-stack-2015/cms.png'></center>
+
+Our family of mobile applications includes [Artsy for iOS](http://iphone.artsy.net), which is a hybrid app written in Objective-C, and a [bidding kiosk](https://github.com/artsy/eidolon), written in Swift. Both are open-source [here](https://github.com/artsy/eigen) and [here](https://github.com/artsy/eidolon).
+
+<center><img src='/images/2015-03-23-artsy-technology-stack-2015/folio.jpg'></center>
+
+A lot of data, including the artwork Similarity graph that powers the Art Genome Project, is processed offline by a generic job engine, written in Ruby. We take data snapshots from MongoDB, run jobs on the data and export data back to the database. Other recently rewritten services include image processing, which creates thumbnails, image tiles for deep zoom and watermarks high quality JPEGs. Several new applications use [PosgreSQL](http://www.postgresql.org).
+
+We use [Statsd](https://github.com/etsy/statsd) and [Graphite](http://graphite.wikidot.com) for tracking high volume events. We recently began doing analytics with [Snowplow](https://github.com/snowplow/snowplow) and data warehousing in [AWS Redshift](http://aws.amazon.com/redshift), slowly moving away from [Chartio](https://chartio.com) and [Keen](https://keen.io). We store user actions via [Segment](https://segment.com) with [Google Analytics](http://www.google.com/analytics) and [MixPanel](https://mixpanel.com). It's fairly common to find a non-Engineer at Artsy in a Rails console directly querying data.
```

Comment:
pps, dunno if its worth mentioning, but we are moving from R to ipython notebooks for analysis on Platform team